### PR TITLE
Add informational views

### PR DESCRIPTION
**Pull request by cevian**
Add views about metrics and labels.

### CODE REVIEW

File: docs/sql_schema.md [hunk 1]
```diff
@@ -112,6 +112,48 @@ Example query to look at all the series:
...
+the following columns
+
+- id
+- metric_name
+- table_name - name of the tables and views in the `prom_metric` and `prom_series`
+  schemas
+- retention_period - the length of time data in this table will be kept
+- label_keys - an array of the label keys used with this metric
+- size - disk space used for storing the metric
+- compression_ratio - the compression ratio achieved for this metric
```
Comment (JLockerman) [inline docs/sql_schema.md:127]: is this `compressed_size / uncompressed_size`? if so we should say it

Comment (cevian) [reply docs/sql_schema.md:127]: 1 - (compressed_size / uncompressed_size)

Comment (davidkohn88) [reply docs/sql_schema.md:127]: Maybe just give them compressed bytes and uncompressed bytes and let them figure out how they want to deal with that themselves?

Comment (davidkohn88) [reply docs/sql_schema.md:127]: It's also very useful in that case to be able to sum sizes etc, because otherwise, with the text repr, it gets really hard

Comment (cevian) [reply docs/sql_schema.md]: I added the description but don't think that giving raw numbers is necessary. Most people only care about ratio and people can always get more info from hypertable_information view.

**Pull request merged**